### PR TITLE
Fixes trackers being limited and also upgrades some stuff

### PR DESCRIPTION
**Pull request by chrisnojima**
So while debugging tracker popups i decided to update some deps (hoping that'd fix things)

### CODE REVIEW

File: desktop/app/index.js [hunk 1]
```diff
@@ -11,6 +11,14 @@ import storeHelper from './store-helper'
 import mainWindow from './main-window'
 import helpHelper from './help-helper'
 
+// MUST do this else we get limited by simultaneous hot reload event streams
+app.commandLine.appendSwitch('ignore-connections-limit', 'localhost')
```
Comment (chrisnojima) [inline desktop/app/index.js]: this was causing the issue. we'd have all these hot reload event streams staying open which starved our connections to localhost

File: desktop/app/index.js [hunk 2]
```diff
@@ -11,6 +11,14 @@ import storeHelper from './store-helper'
...
+
+if (__DEV__) { // eslint-disable-line no-undef
+  app.commandLine.appendSwitch('enable-logging')
```
Comment (chrisnojima) [inline desktop/app/index.js]: might as well add some logging stuff

File: desktop/package.json [hunk 3]
```diff
@@ -7,10 +7,10 @@
     "start": "npm run build-dev && npm run start-cold",
     "start-hot": "HOT=true NODE_ENV=development node client.js",
     "start-cold": "NODE_ENV=development electron ./dist/main.bundle.js",
-    "build-dev": "NODE_ENV=development NO_SERVER=true DEBUG='express:*' node server.js",
+    "build-dev": "NODE_ENV=production NO_SERVER=true DEBUG='express:*' node server.js",
```
Comment (chrisnojima) [inline desktop/package.json]: we no longer use the NODE_ENV to determine what happens, now its based on development/prod bundles and not the calling context

File: desktop/package.json [hunk 4]
```diff
@@ -22,38 +22,34 @@
     "babel-core": "^5.8.34",
     "babel-loader": "^5.4.0",
     "babel-runtime": "^5.8.34",
-    "core-decorators": "^0.9.1",
```
Comment (chrisnojima) [inline desktop/package.json]: removed a bunch of stuff we don't use anymore. updated stuff we do

File: desktop/renderer/index.js [hunk 5]
```diff
@@ -6,8 +6,6 @@ import {Provider} from 'react-redux'
 import configureStore from '../../react-native/react/store/configure-store'
 import Nav from '../../react-native/react/nav'
 import injectTapEventPlugin from 'react-tap-event-plugin'
-import {reduxDevToolsSelect} from '../../react-native/react/local-debug.desktop'
```
Comment (chrisnojima) [inline desktop/renderer/index.js]: redux devtools changed slightly. its faster and better now (ctrl+h to show, ctrl+q to move to various parts of the screen)

File: react-native/react/router/meta-navigator.render.desktop.js [hunk 6]
```diff
@@ -35,7 +35,9 @@ const styles = {
     position: 'absolute',
     top: 0,
     right: 0,
-    backgroundColor: '#cccccc'
+    backgroundColor: '#cccccc',
```
Comment (chrisnojima) [inline react-native/react/router/meta-navigator.render.desktop.js]: fixes back button being underneath stuff sometimes

File: react-native/react/search/index.desktop.js [hunk 7]
```diff
@@ -80,20 +80,21 @@ export default class Search extends Component {
   }
 
   render () {
```
Comment (chrisnojima) [inline react-native/react/search/index.desktop.js]: This isn't a real implementation so let's just comment this out. eventually it'll use material-ui and not this one-off library

**Comment by chrisnojima** [comment]
@keybase/react-hackers

**Comment by MarcoPolo** [comment]
How does this fix the tracker popups?

File: desktop/app/index.js (hunk 1 above)
Comment (chrisnojima) [inline desktop/app/index.js]: @MarcoPolo ^

**Comment by MarcoPolo** [comment]
:+1:

**Pull request merged**